pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -102,6 +102,11 @@ type Config struct {
 	dikastesEnabled bool
 	envoyConfigMap  *corev1.ConfigMap
 
+	// envoy user-configurable overrides
+	UseRemoteAddressXFF bool
+	NumTrustedHopsXFF   int32
+	SkipAppendXFF       bool
```

Comment:
Sorry one more very minor thing. We're not making use of this field anymore so please remove it.